In [1]:
import numpy as np
import pandas as pd
import statsmodels.formula.api as smf

# Не показывать FutureWarnings
import warnings
warnings.simplefilter(action='ignore', category=FutureWarning)

In [2]:
# импорт данных
df = pd.read_csv('Labour.csv')
df

capital  labour     output       wage
0     2.606563     184   9.250759  43.080307
1     1.323237      91   3.664310  27.780016
2    22.093692     426  28.781516  44.467748
3    10.737851      72   4.124642  39.734710
4     1.161365      46   2.890150  34.650709
..         ...     ...        ...        ...
564   2.625403      20   1.424376  33.477545
565   1.276386      61   2.109048  26.300732
566   1.953869     117   6.241870  41.153979
567   1.318527      46   7.902237  66.720139
568   2.832391     161   6.804085  36.618435

[569 rows x 4 columns]

In [3]:
mod = smf.ols(formula = 'np.log(output)~1+np.log(capital)+labour+wage' , data = df)
res = mod.fit()

In [4]:
res.params

Intercept          0.577725
np.log(capital)    0.442294
labour             0.000450
wage               0.016260
dtype: float64

$$ log(output) = \beta_0 + \beta_1 log(capital) + \beta_2 labour + \beta_3 wage + u$$

Оценка регрессии: 
зависимая перменная - output 
регрессоры - capital, labour, wage
output - выпуск (в миллионах евро)
capital - капитал (в миллионах евро)
labour - число сотрудников 
wage - зарпалата на одного сотрудника (в тыс евро)

При увеличении значения capital на единицу при прочих равных, переменная output изменится на 0.44
При увеличении значения labour на единицу при прочих равных, переменная output изменится на 0.05 (коэффициент умноженный на 100)
При увеличении значения wage на единицу при прочих равных, переменная output изменится на 1.63 (коэффициент умноженный на 100)